### ISSUE TITLE
phpDocumentor does not finalize the output

### ISSUE THREAD

**Issue by Dagobert57** [phpDocumentor/phpDocumentor2#2090]
Hello,
I installed phpDocumentor 2.9.0 via pear on two machines (debian 9.9 where also the webserver nginx runs and on my notebook with ubuntu 18.04) and want now to learn how it works. phpDocumentor was correctly installed and is apparently working when I type the command `phpdoc.` I found a [tutorial](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_phpDocumentor.quickstart.pkg.html) that explains the bases of phpDocumentor:

On this site there are three samples.
1. [sample1.php](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_sample1.pkg.html)
2. [sample2.php](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_sample2.pkg.html)
3. [sample3.php](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_sample3.pkg.html)

I copied the source code of these three samples into a directory and did
`phpdoc run -f sample1.php -t docs` (I learned that the option `-o` does no longer exist in version 2 of phpdoc).

Now phpdoc begins to work and produces the following [output.txt](https://github.com/phpDocumentor/phpDocumentor2/files/3219727/output.txt).
In the newly created directory "docs" there is no file like "index.html", only the following directories:
`css font images js phpdoc-cache-2a phpdoc-cache-2e`
It seems that phpdoc does not finalize the output and does not write its results in a html file.
I tested the samples also with the version 1.4.4 of phpdoc. This version creates html files in the directory "docs" but they are all empty.

Could someone please give me a hint? Thank you very much.

**Comment by mvriel** [comment]
Hello @Dagobert57,

You are correct that no files are created. An error seems to occur when phpDocumentor starts to build the HTML files in its Markdown library.

Commonly these errors occur due to issues with PHP versions. Which version are you running? And do you experience the same issues with phpDocumentor 3 alpha 3 or the latest version on develop?

**Comment by Dagobert57** [comment]
Hello mvriel,

thank you very much for your reply. I used on the NAS with debian 9.9 php version 7.0.33 and on my notebook with ubuntu 18.04 php 7.2.19.
I tested on both machines the phar file version 2.9.0 with the following command:

`php phpDocumentor.phar -f sample1.php -t docs`

That works without any problems.

But when I use the pear version 2.9.0 on both machines I get the result that I described in my article above - it does not finalize the output. It was not really clear for me how easy it is to use the phar version.

When I try to use the latest version Release v3.0.0-alpha.3 via phar I have the following result on both machines:
[result.txt](https://github.com/phpDocumentor/phpDocumentor2/files/3311833/result.txt)

But for me it does not matter, I'm glad about having a running system with the version 2.9.0.
